resources/styles/molecules/accordion/BpAccordion.vue

```diff
@@ -1,30 +1,30 @@
 <template>
-    <div :class="[block, isOpen ? '-open' : '-closed' ]">
-        <header>
-            <div
-                :id="`${id}-control`"
-                :class="`${block}__${headerElement}`"
-                :aria-expanded="`${isOpen}`"
-                :aria-controls="id"
-                tabindex="0"
-                role="button"
-                @click="open"
-                @keyup.enter="open"
-            >
-                <slot name="heading" />
-                <slot name="icon">
-                    <svg :class="`${block}__icon`" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none">
-                        <polyline points="6 9 12 15 18 9" />
-                    </svg>
-                </slot>
-            </div>
-        </header>
-        <transition
+    <div :class="[block, isOpen ? '-open' : '-closed']">
+        <button
+            :id="`${id}-control`"
+            :class="`${block}__${headerElement}`"
+            :aria-expanded="`${isOpen}`"
+            :aria-controls="id"
+            @click="open"
+            @keyup.enter="open"
+        >
+            <slot name="heading" />
+            <slot name="icon">
+                <svg
+                    :class="`${block}__icon`"
```

Comment:
Small nit-pick. But I would add a prop for the icon element name. I realize it's a slot default and will most likely get overridden, but good to have.